### DISCUSSION TITLE
info/KubeJS/Q&A/error

### DISCUSSION THREAD

**Discussion by giscus** [PickAID/CrychicDoc#92] [Announcements]
# info/KubeJS/Q&A/error

一个包含 Minecraft 开发文档的网站。

https://docs.mihono.cn/zh/info/KubeJS/Q&A/error

**Comment by M1hono** [comment]
```js
TypeError: redeclaration of var variable
```
检查有没有在循环或函数中使用了const新建变量，尝试将这些const换为let，如果还不行，就将所有的const都换为let。
